.github/dependabot.yml

```diff
@@ -1,11 +1,23 @@
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "/"
+  - package-ecosystem: npm
+    directory: /
     schedule:
-      interval: "daily"
-      time: "04:00"
+      interval: daily
+      time: '04:00'
     commit-message:
       prefix: fix
       prefix-development: chore
       include: scope
+    labels:
+      - dependabot
+    groups:
+      security-updates:
+        applies-to: security-updates
+        update-types:
+          - patch
+    ignore:
+      - dependency-name: '*'
```

Comment:
@edodusi wouldn't the `ignore` statement prevent the releases from working properly? Because it'll ignore also the updates coming from `@storyblok/js`.

If that's the case, can we set it to "ignore all dependencies except `@storyblok/js`"? cc @alvarosabu

Reply:
@alexjoverm look at the line after, this is to tell dependabot to ignore all `minor` and `major` versions of all packages, and focus only on `patch`.

This is just for security updates, I should include a new rule for storyblok packages

Reply:
@edodusi in the way is set up, on the lines above (the `groups` statement) it where it groups the security updates. But the `ignore` applies to all minors and majors, not only security ones. Correct?

If that's the case, if in `@storyblok/js` we publish a new minor release, wouldn't that be ignored by this pkg? If the idea is to have a separate dependabot config for storyblok package, is it included in this PR?

Reply:
yes exactly, because the desired outcome would be that dependabot only opens auto-approved PR with grouped patches (we can decide to auto-merge them), and we control manually minors and majors because sometime there are changes that alter some stuff even in minor updates and it's better we do that manually

For storyblok packages we should have specific rules, because maybe we want patches and minors to be automatically merged

Reply:
_"because maybe we want patches and minors to be automatically merged"_ -> Exactly, we need it as that's how the whole release system works currently. Can we add them?

About security updates, makes total sense 👍